docs/config/options.md

```diff
@@ -37,6 +37,22 @@ The paths are relative to the configuration file and should point to a lua(u) fi
 
 <CodeBlock :code="outputExample" />
 
+## `event_handling`
+
+Determines the default way your code receives events from Zap. With signals, Zap pushes events as it receives them to your listener functions. With polling, Zap stores events in queues when they're received, and you pull events from those queues when you want them.
+
+### Default
+
+<CodeBlock code = 'opt event_handling = "signal"' />
+
+### Example
+<script setup lang="luau">
+-- Example of using a Zap polling event on the server.
+for player, payload in zap.my_event do
+	print(player, payload)
+end
+</script>
```

Comment:
`<script setup` is actually a `vue` thing where it runs code at compile time.
Up on ~ L20 add something like
```ts
const handleExample = `-- Example of using a Zap polling event on the server.
for player, payload in zap.my_event do
	print(player, payload)
end`
```
and then here you can replace it with
```suggestion

<CodeBlock :code="handleExample" />
```